src/js/components/Widgets/ReadMore.js

```diff
@@ -41,7 +41,7 @@ export default class ReadMore extends Component {
           num_of_lines = 3;
         }
         if (link_text === undefined) {
-          link_text = "More";
+          link_text = "SignIn";
```

Comment:
Hi Steve, it looks like there was a search-and-replace from "More" to "SignIn" that wasn't what you had planned.